spec/support/shared_examples/cli/new.rb

```diff
@@ -44,7 +44,6 @@
       # lib/<project>.rb
       #
       expect("lib/#{project}.rb").to have_file_content <<-END
```

Comment:
can this be changed to just `""` (or `"\n"?`) instead of using the HEREDOC?

Also, do we need this file at all, if it's blank? If so, could we put a comment in here saying something like 
```ruby
# This file is intentionally blank.
```